### ISSUE TITLE
Good but would need update

### ISSUE THREAD

**Issue by solomon-ey** [thelinuxchoice/shellphish#56]
first of all i want to give kudos to @thelinuxchoice for the rare tool. nice work you have done here but i want to suggest and update for the tool which i notice.

1. the login page links are olds and will need updates.
2. there should be a way to edit the shellphish.sh pages
3. if possible to make it a true login

I'm sure with this suggestion the tool will be one of the best ever written.

Thank you

**Comment by megajuacker10** [comment]
I agree with you friend. An easy way to edit the interfaces, since Facebook, for example, is a total fiasco.

**Comment by megajuacker10** [comment]
I agree with you friend. An easy way to edit the interfaces, since Facebook, for example, is a total fiasco.
